prime-angular-frontend/src/app/modules/Test.ts

```diff
@@ -0,0 +1,58 @@
+export class TestingStuff {
```

Comment:
What is all this?

Reply:
This will be deleted as in the commit message. I just wanted to add useless code without adding any unit test to see why the diff coverage is always 100%